src/identity/manager.rs

```diff
@@ -122,6 +123,17 @@ impl SecretManager<CaClient> {
     }
 }
 
+impl SecretManager<CustomSigner> {
+    pub fn origin(cfg: crate::config::Config) -> Result<Self, Error> {
+        let cache: HashMap<Identity, watch::Receiver<Option<tls::Certs>>> = Default::default();
```

Comment:
How often is the cache refreshed?

Reply:
The cache is refreshed after half of the cert life is over.

Reply:
Can you point me to the code that does it?

Reply:
https://github.com/istio/ztunnel/blob/master/src/identity/manager.rs#L233
https://github.com/istio/ztunnel/blob/master/src/tls/boring.rs#L165